workers/unity/Packages/com.improbable.gdk.gameobjectcreation/IEntityGameObjectCreator.cs

```diff
@@ -22,6 +23,6 @@ public interface IEntityGameObjectCreator
         /// <param name="linkedGameObject">
         ///     The GameObject linked to the entity, or null if there is none.
         /// </param>
-        void OnEntityRemoved(SpatialOSEntity entity, GameObject linkedGameObject);
+        void OnEntityRemoved(SpatialEntityId entityId, GameObject linkedGameObject);
```

Comment:
This can just be an EntityId. Inside the SpatialOSEntity it's just a regular EntityId.

Reply:
Also we have mismatched names here. Generally you shouldn't rename arguments between the interface and the implementation.

Reply:
yeah, that happened by accident. reverting it